label_studio/webhooks/serializers_for_hooks.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import serializers
+
+from projects.models import Project
+from tasks.models import Task, Annotation
+# SERIALIZERS FOR WEBHOOKS
+
+
+class OnlyIDWebhookSerializer(serializers.Serializer):
+    id = serializers.IntegerField()
+
+    class Meta:
+        fields: ('id',)
+
+
+class ProjectWebhookSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Project
+        fields = '__all__'
+
+
+class TaskWebhookSerializer(serializers.ModelSerializer):
```

Comment:
We need to copy logic from this serializer ExportDataSerializer: https://github.com/heartexlabs/label-studio/blob/master/label_studio/data_export/serializers.py#L26